locales/en.default.schema.json

```
@@ -1729,6 +1729,100 @@
         "name": "Email signup"
       }
     },
+    "email-signup-banner": {
+      "name": "Email signup banner",
+      "settings": {
+        "paragraph": {
+          "content": "Each email subscription creates a customer account. [Learn more](https://help.shopify.com/en/manual/customers)"
+        },
+        "image": {
+          "label": "Background image"
+        },
+        "show_background_image": {
+          "label": "Show background image"
+        },
+        "desktop_text_box_position": {
+          "options__1": {
+            "label": "Top"
+          },
+          "options__2": {
+            "label": "Center"
+          },
+          "options__3": {
+            "label": "Bottom"
+          },
+          "label": "Desktop text position"
+        },
+        "show_text_box": {
+          "label": "Show text box on desktop"
+        },
+        "image_overlay_opacity": {
+          "label": "Image overlay opacity"
+        },
+        "color_scheme": {
+          "options__1": {
+            "label": "Accent 1"
+          },
+          "options__2": {
+            "label": "Accent 2"
+          },
+          "options__3": {
+            "label": "Background 1"
+          },
+          "options__4": {
+            "label": "Background 2"
+          },
+          "options__5": {
+            "label": "Inverse"
+          },
+          "label": "Color scheme",
+          "info": "Visible when text box displayed"
+        },
+        "show_text_below": {
+          "label": "Show text below image on mobile"
+        },
+        "adapt_height_image": {
+          "label": "Adapt banner height to image"
+        },
+        "image_height": {
+          "label": "Image banner height",
+          "options__1": {
+            "label": "Small"
+          },
+          "options__2": {
+            "label": "Medium"
+          },
+          "options__3": {
+            "label": "Large"
+          },
+          "info": "For best results, use an image with a 16:9 aspect ratio. [Learn more](https://help.shopify.com/en/manual/shopify-admin/productivity-tools/image-editor#understanding-image-aspect-ratio)"
+        }
```

```suggestion
        }
```
Is there missing a coma here?

---

That is the last item so no coma needed :)